test/nn/simplicial/test_dist2cycle.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for Dist2Cycke Model."""
+import numpy as np
+import numpy.linalg as npla
+import torch
+from toponetx.classes import SimplicialComplex as sc
+
+from topomodelx.nn.simplicial.dist2cycle import Dist2Cycle
+
+
+class TestDist2Cycle:
+    """Unit tests for the Dist2Cycle model class."""
+
+    def test_forward(self):
+        """Test the forward method of Dist2Cycle."""
+        edge_set = [[1, 2], [1, 3], [2, 5], [3, 5]]
+        face_set = [[2, 3, 4], [2, 4, 5]]
+
+        torch.manual_seed(42)
+        ex2_sc = sc(edge_set + face_set)
+        ld = ex2_sc.down_laplacian_matrix(rank=1).todense()
+        A = ex2_sc.adjacency_matrix(rank=1).todense()
+        L_tilde_pinv = npla.pinv(ld + np.eye(ld.shape[0]))  # test inverse
```

Comment:
Ditto: find better names by: (1) reading my comments above naming above, (2) looking in the codebase of TopoModelX to observe which conventions are chosen, and respect the same ones.